$OpenBSD: patch-libgnomecups_gnome-cups-printer_h,v 1.3 2009/04/10 16:17:49 ajacoutot Exp $

Fix PPD retrieval for printers picked up with cups browsing.

--- libgnomecups/gnome-cups-printer.h.orig	Tue Jan 29 23:50:53 2008
+++ libgnomecups/gnome-cups-printer.h	Tue Jan 29 23:51:09 2008
@@ -188,6 +188,7 @@ void                    gnome_cups_ppd_file_release   
 /* Private */
 void _gnome_cups_printer_init (void);
 gchar *_gnome_cups_printer_get_host (GnomeCupsPrinter *printer);
+gchar *_gnome_cups_printer_get_ppd_host (GnomeCupsPrinter *printer);
 
 G_END_DECLS
 	
